### ISSUE TITLE
Loss

### ISSUE THREAD

**Issue by BingCS** [jenspetersen/gqn-pytorch#3]
Hi,

Many thanks for your contribution on implementing GQN. 

Recently, I implemented GQN. After the training on rooms_ring_camera, I found that the loss will converge to around 6950 after 200K iterations, which means after the sigma decrease to 0.7, the loss will maintain at a stable level. But in their original paper, the final loss is just 6.1.  I don’t understand the where is the problem. Do you have the same problem?  Many thanks in advance!

Looking forward to your reply!

**Comment by jenspetersen** [comment]
To be completely honest, I don't know. a) I've only ever used the Shepard-Metzler-5 dataset for GQN, b) I can't look up my loss values, because I didn't keep my experiment outputs and c) I don't really have capacities to retrain. That being said, that loss value sounds too high... is the model output any good? I'm guessing it's your reconstruction loss that's so high? What does your KL look like? Maybe it's close to 0 and you've encountered the good old posterior collapse? :D Although at 200K iterations that seems a little late, but who knows. The paper proposes to slowly increase the KL weight over a number of episodes (e.g. from 0.05 to 1 over 100K), do you do that?
